TRADEMARKS.md

```diff
@@ -68,6 +69,8 @@ The Commonhaus Foundation uses the following unregistered trademarks:
     <img src="https://github.com/easymock/objenesis/blob/master/website/site/resources/objenesis-logo.png?raw=true" alt="" height="30" />
 - Quarkus Logo &trade; [^1]  
     <img src="https://github.com/quarkusio/quarkus/blob/main/core/deployment/src/main/resources/META-INF/branding/logo.png?raw=true" alt="" height="30" />
+- RESTEasy Logo &trade; [^1]  
+  <img src="https://github.com/resteasy/resteasy.dev/blob/main/img/logo/resteasy_logo_600x.png?raw=true" alt="" height="30" />
```

Comment:
Nitpick:

s/alt=""/alt="RESTEasy logo"

This is not a decorative image so it should have a proper alt description.

Reply:
That would make this the outlier from all the others which have `alt=""`.

Reply:
A correct outlier that is :-) It's definitely not the scope of this PR to fix this for others.

Reply:
I felt that putting the wordmark version of the logo in as the alt tag is also redundant. It is in a logo section. What they are for is obvious. Wordmark is already listed above.